astropy/modeling/functional_models.py

```diff
@@ -1049,10 +1049,10 @@ class Voigt1D(Fittable1DModel):
         plt.show()
     """
 
-    x_0 = Parameter(default=0)
-    amplitude_L = Parameter(default=1)
-    fwhm_L = Parameter(default=2/np.pi)
-    fwhm_G = Parameter(default=np.log(2))
+    x_0 = Parameter(default=0, description="Position of the peak")
+    amplitude_L = Parameter(default=1, description="The Lorentzian amplitude")
```

Comment:
```suggestion
    amplitude_L = Parameter(default=1, description=
                            "Lorentzian amplitude (peak value of the associated Lorentz function)")
```
Note that I have updated some of the docstrings for the Gauss and Lorentz-related functions in #11177; if the `description` strings here are in sync, I can merge them when the time has come.